manual/manual_compiler.rst

```diff
@@ -718,6 +718,54 @@ For variational algorithms, the prominent benefit of defining a :py:class:`Circu
 
 .. note:: Every :py:class:`Backend` requires :py:class:`NoSymbolsPredicate`, so it is necessary to instantiate all symbols before running a :py:class:`Circuit`.
 
+User-defined Passes
+===================
+
+We have already seen that pytket allows users to combine passes in a desired order using :py:class:`SequencePass`. An addtional feature is the :py:class:`CustomPass` which allows users to define their own custom circuit transformation using pytket.
+The :py:class:`CustomPass` class accepts a ``transform`` parameter, a python function that takes a :py:class:`Circuit` as input and returns a :py:class:`Circuit` as output. 
+
+We will show how to use :py:class:`CustomPass` by defining a simple tranfromation that replaces any Pauli Z gate in the :py:class:`Circuit` with a Hadamard gate, Pauli X gate, Hadamard gate chain.
```

Comment:
tranfromation -> transformation

Reply:
Done. I managed to deactivate spellchecker on VScode by accident lol.